### ISSUE TITLE
Update, I cannot...

### ISSUE THREAD

**Issue by annasmithforos** [dave-theunsub/clamtk-gnome#19]
Hello. I have clamtk-gnome installed but have never been able to get it to show up in the menu. In Synaptic the installed files are /.
/usr
/usr/share
/usr/share/doc
/usr/share/doc/clamtk-gnome
/usr/share/doc/clamtk-gnome/README_clamtk-gnome
/usr/share/doc/clamtk-gnome/changelog.Debian.gz
/usr/share/doc/clamtk-gnome/copyright
/usr/share/nautilus-python
/usr/share/nautilus-python/extensions
/usr/share/nautilus-python/extensions/clamtk-gnome.py
/usr/share/pixmaps
/usr/share/pixmaps/clamtk-gnome.png
 Now I just uploaded clamtk 6.15 1 and I want to upload the corresponding plugin, but I don't see where to download the deb or rpm package from. It's probably my ignorance. You can help me?
I'm using ubuntu mate 22.10.
![Captura de pantalla -2023-03-30 17-15-45](https://user-images.githubusercontent.com/127437263/228883686-3ce80f76-223a-4299-b666-c287fa60d1c3.png)
Thank you.

**Comment by dave-theunsub** [comment]
Hi @annasmithforos ,

The clamtk-gnome package is available here:  
https://github.com/dave-theunsub/clamtk-gnome/releases/tag/v6.15

It's not released in any repositories that I'm aware of.  But anyway, you should be able to download, double-click (or use dpkg -i using CLI), and go.

See if that helps.

respectfully,
Dave M

**Comment by annasmithforos** [comment]
Yes, it works, thank you very much. The problem is that I couldn't find it. You can close issue.
Best regards.
